### PR TITLE
Resurrected 92 fetch saved frames

### PR DESCRIPTION
**Pull request by shawnkendrick**
Updated to master.

### CODE REVIEW

File: app/state/stateSvc.js [hunk 1]
```diff
@@ -488,6 +490,21 @@ function stateSvc(
     return svc.config.storypins;
   };
 
+  svc.save_storyframes = storyframes => {
```
Comment (Coop56) [inline app/state/stateSvc.js]: This function name should use JS camelcase syntax.

File: app/state/stateSvc.js [hunk 2]
```diff
@@ -488,6 +490,21 @@ function stateSvc(
...
+    svc.config.storyframes = storyframes;
+    console.log('save_storyframes ', storyframes);
+  };
+
+  svc.get_storyframes = () => {
```
Comment (Coop56) [inline app/state/stateSvc.js]: This function name should use JS camelcase syntax.

File: app/state/stateSvc.js [hunk 3]
```diff
@@ -190,6 +190,7 @@ function stateSvc(
       // Initialize empty arrays for storypins
       svc.getConfig().storypins = [[]];
       // TODO: Do the same for storyframes
+      // svc.getConfig() = [[]];
```
Comment (Coop56) [inline app/state/stateSvc.js]: Can this commented out code go or is it meant to be used?

Comment (shawnkendrick) [reply app/state/stateSvc.js]: Yes, removed.

File: app/state/stateSvc.js (hunk 2 above)
Comment (shawnkendrick) [reply app/state/stateSvc.js]: Converted to camelcase.

File: app/state/stateSvc.js (hunk 1 above)
Comment (shawnkendrick) [reply app/state/stateSvc.js]: Converted to camelcase.

**Pull request merged**